app/controllers/educators_controller.rb

```diff
@@ -1,5 +1,6 @@
 class EducatorsController < ApplicationController
   # Authentication by default inherited from ApplicationController.
+  TIME_INTERVAL = Date.today - 2400 # TODO: change to 30 before merging
```

Comment:
@gabivoicu What do you think about making the variable name for this constant more specific to make it clear that it's the time interval for the My Notes feature?